src/components/page/JoinNow.svelte

```diff
@@ -133,16 +145,32 @@
             />
           </label>
         </div>
-        <div class="input-block">
+        <div class="input-block" on:blur={() => showEmailDropdown = false}>
           <label
             >Email ¹<input
               type="email"
               name="email"
               required
+              autocomplete="off"
+              on:input={watchEmailInput}
+              on:focus={watchEmailInput}
               bind:this={emailInput}
             /></label
           >
+
+          {#if showEmailDropdown}
+            <div class="email-selector">
+              <div on:click={() => appendEmail('@gmail.com')} class="email-selector__item">@gmail.com</div>
+              <div on:click={() => appendEmail('@hotmail.com')} class="email-selector__item">@hotmail.com</div>
+              <div on:click={() => appendEmail('@yahoo.com')} class="email-selector__item">@yahoo.com</div>
+              <div on:click={() => appendEmail('@outlook.com')} class="email-selector__item">@outlook.com</div>
+            </div>
```

Comment:
not super important but you could probs refactor by doing something like:
`{#each emailDomains as domain}
      <div on:click={() => appendEmail(domain)} class="email-selector__item">{domain}</div>
    {/each}` 
then define the domains in an array, just so its easy to maintain